bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/internal/db/JdbcPostgresqlDAO.java

```diff
@@ -226,7 +269,7 @@ public void doAlterTableColumn(String tableName, String columnName, String colum
     @Override
     public void doStoreItemValue(Item item, State itemState, ItemVO vo) throws JdbcSQLException {
         ItemVO storedVO = storeItemValueProvider(item, itemState, vo);
-        String sql = StringUtilsExt.replaceArrayMerge(sqlInsertItemValue,
+        String sql = StringUtilsExt.replaceArrayMerge(this.sqlInsertItemValue,
```

Comment:
Please revert:
```suggestion
        String sql = StringUtilsExt.replaceArrayMerge(sqlInsertItemValue,
```

We generally only use `this.` prefixes when necessary.

Reply:
I used the `this.` because otherwise it uses the default implementation query.

Reply:
It will be implicitly considered as `this.sqlInsertItemValue`. Otherwise you would have to reference it by `super.sqlInsertItemValue`.